examples/nuclei_image_segmentation/README.md

```diff
@@ -0,0 +1,13 @@
+# Nuclei Image Segmentation With UNets
+
+## Setup
+
+1. Install dependencies: `pip install -r requirements.txt`
+1. Follow [Kaggle's installation and configuration documentation](https://github.com/Kaggle/kaggle-api#installation) to install and configure the kaggle cli
+1. Go to Kaggle's [2018 Data Science Bowl Competition](https://www.kaggle.com/c/data-science-bowl-2018), click on "Late Submission" and accept the terms and conditions to get access to the data.
+1. Run `./dl_extract_prepare.sh` to download, extract and prepare the data.
```

Comment:
For me this fails on `from PIL import Image`. This again comes to the requirements installation. Neither Pillow nor torchvision is in `requirements.txt`, so novice user may be left puzzled what's going on and why something is missing after the dependencies installation step has succeeded.